Description: Fix the compatibility with the version of Gradle in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/settings.gradle
+++ b/settings.gradle
@@ -10,4 +10,3 @@
     assert project.buildFile.isFile()
 }
 
-enableFeaturePreview('STABLE_PUBLISHING')
--- a/build.gradle
+++ b/build.gradle
@@ -83,7 +83,6 @@
 
 wrapper {
     gradleVersion = '4.9'
-    distributionSha256Sum = 'e66e69dce8173dd2004b39ba93586a184628bc6c28461bc771d6835f7f9b0d28'
 }
 
 
